Fred2/EpitopeSelection/OptiTope.py

```diff
@@ -135,7 +135,7 @@ def __init__(self, results,  threshold=None, k=10, solver="glpk", verbosity=0):
             p = tup[0]
             seq = str(p)
             peps[seq] = p
-            for a, s in itr.izip(res_df.columns, tup[1:]):
+            for a, s in zip(res_df.columns, tup[1:]):
```

Comment:
AttributeError: module 'itertools' has no attribute 'izip'
itertools.izip not available in python3, use built in zip
https://stackoverflow.com/questions/32261698/my-idle-does-not-recognize-itertools-izip-as-a-function